sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuExpressions.scala

```diff
@@ -89,6 +89,18 @@ object GpuExpressionsUtils extends Arm {
    */
   def columnarEvalToColumn(expr: Expression, batch: ColumnarBatch): GpuColumnVector =
     resolveColumnVector(expr.columnarEval(batch), batch.numRows, expr.dataType)
+
+  /**
+   * Extract the GpuLiteral
+   * @param exp the input expression to be extracted
+   * @return an optional GpuLiteral
+   */
+  @scala.annotation.tailrec
+  def extractGpuLit(exp: Expression): Option[GpuLiteral] = exp match {
```

Comment:
nit: This really feels like it should be in an object instead of a class (static method instead of non-static)

Reply:
It's in the `object GpuExpressionsUtils`.